### PR TITLE
[BLE] Implement FIDO Credentials tab

### PR DESCRIPTION
**Pull request by deXol**
FIDO Credentials page:
image
After entering FIDO management:
image
After removing a FIDO credential:
image

### CODE REVIEW

File: src/CredentialModel.cpp [hunk 1]
```diff
@@ -244,9 +226,34 @@ void CredentialModel::load(const QJsonArray &json)
             // Update login item address
             pLoginItem->setAddress(bAddress);
 
-            // Update login favorite
-            int iFavorite = cnode["favorite"].toInt();
-            pLoginItem->setFavorite(iFavorite);
+            if (isFido)
+            {
+                pLoginItem->setCategory(0);
```
Comment (limpkin) [inline src/CredentialModel.cpp:231]: what's that for? :)

File: src/MPDevice.cpp [hunk 2]
```diff
@@ -903,21 +906,23 @@ void MPDevice::startMemMgmtMode(bool wantData,
     startMmmJob->setTimeout(15000); //We need a big timeout here in case user enter a wrong pin code
     jobs->append(startMmmJob);
 
+    //Fetch credential when not data and not fido wanted
+    bool wantCredentials = !wantData && !wantFido;
```
Comment (limpkin) [inline src/MPDevice.cpp:910]: very nice

**Review by limpkin — COMMENTED** [review]
very nice code! see my comments. Also:
- on an empty DB I get both "Discard deletions" & "Save & Quit" buttons... should we only have "quit" like for standard MMM?

File: src/CredentialModel.cpp (hunk 1 above)
Comment (deXol) [reply src/CredentialModel.cpp:231]: Without that category value is not initialized and it had have a value >0 so the category icon was displayed.

**Pull request merged**